api/organisations/templates/organisations/api_usage_notification.txt

```diff
@@ -0,0 +1,7 @@
+Hi there,
+
+API usage for {{ organisation.name }} has reached {{ matched_threshold }} within the current subscription period. Please consider upgrading your organisations account limits.
```

Comment:
We might need to be more aggressive for the higher thresholds (100 / 120) but for now this is probably fine.

Reply:
I decided that we should implement these higher threshold emails now, so I've worked them in (in both html and txt formats).

Reply:
Nice ok 👍